src/main/java/com/es/phoneshop/model/entity/cart/Cart.java

```diff
@@ -0,0 +1,21 @@
+package com.es.phoneshop.model.entity.cart;
```

Comment:
Put all entity classes in **entity** package directly without sub-packages like **cart, latestProductQueue**